tests/cpp/test_sdpa_node.cpp

```diff
@@ -610,4 +634,91 @@ TEST_F(SDPATest, AttnProgram) {
   EXPECT_TRUE(at::allclose(out[0], expected_out));
 }
 
+TEST_F(SDPATest, AttnFwdBwd) {
+  NVFUSER_TEST_CUDA_ARCH_GUARD(8, 0);
```

Comment:
What's the problem with Volta? `at::scaled_dot_product_attention` doesn't like it?

Reply:
Flash Attention requires Ampere and above.